src/app/forms/custom-sections/letters/letters.component.ts

```diff
@@ -86,6 +99,11 @@ export class LettersComponent implements OnInit, OnDestroy, OnChanges {
     }
 
     if (this.techRecord?.techRecord_statusCode !== StatusCodes.CURRENT) {
+      // if the Technical Record History has current status code return related words to inform users.
+      if (this.checkForCurrentRecordInHistory) {
+        // eslint-disable-next-line max-len
+        return 'Generating letters is not applicable to provisional records, where a current record also exists for a vehicle. Open the current record to generate letters.';
+      }
       return 'Generating letters is only applicable to current technical records.';
```

Comment:
this message is no longer correct as you are able to do this for provisional records too as part of this work, it should be altered to reflect this behaviour

Reply:
Ahh, this logic is a bit tricky to make sense. The AC of my ticket aims to return this message when the provisional technical record has the current status record in the _Technical Records History_.

Reply:
Yeah i am talking about line 107 'Generating letters is only applicable to current technical records' - shouldn't this now include something about provisionals, id imagine you would only see this for an archived record

Reply:
HAHA, yeah, that is a good shout. I missed this line before. Thank you sooooo much for pointing it out.

Reply:
i wonder if it would be better practice to do
if (provisional) {
  check for current record in history
    return Generating letters is not applicable to provisional records, where....
} else if (archived) {
  return Generating letters is not applicable to archived records.
}
as doing !current makes the type of status ambiguous when it reaches that last return statement

Reply:
Thank you, I will amend it now .